src/fsevents/fsevents_stubs.c

```diff
@@ -90,9 +93,10 @@ static void dune_fsevents_callback(const FSEventStreamRef streamRef,
     CFStringRef cf_path;
 #if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101300
     CFDictionaryRef details = CFArrayGetValueAtIndex(eventPaths, i);
-    cf_path = CFDictionaryGetValue(details, kFSEventStreamEventExtendedDataPathKey);
+    cf_path =
```

Comment:
This hunk looks like just whitespace changes

Reply:
Yes, the auto formatter (the one from ccls) insists on this.

Reply:
I'm confused. Do we enforce this? If that's the case, why wasn't it an issue before? And if that's not the case, does that mean that your local setup is going to reformat all the C files you're editing?

Reply:
We don't, but it's the auto formatter I used for all the stubs I've written - including this one. Might as well keep it consistent in at least on a per file basis.

Reply:
Yes, I also asked about this change which does nothing earlier. But since it does not break anything either, after all…